cmd/istioctl/main.go

```diff
@@ -99,6 +99,7 @@ var (
 			if len(args) != 0 {
 				return fmt.Errorf("create takes no arguments")
 			}
+			c.SilenceUsage = true
```

Comment:
In Cobra, the default is to have Cobra print a command's syntax if it returns an error.

_istioctl_ returns an error if there is a network problem talking to Kubernetes or if the .yaml file is invalid.  By unsetting SilenceUsage Cobra will print the error,  but not the command syntax.